### DISCUSSION TITLE
How to pass theme section file path into APP Block

### DISCUSSION THREAD

**Discussion by ingalesachin7** [Kyon147/laravel-shopify#151] [Q&A]
How to pass theme section file path into  App extension Block?

I want to pass file path from theme section file to block

e.g

{% render 'testliquid' %} - This should be rendered from theme , not from the app block
{% schema %}
{
  "name": "My Block",
  "target": "body",
  "settings": []
}
{% endschema %}

currenlty showing error - Could not find app snippet snippets/testliquid.liquid -->

**Comment by Kyon147** [comment]
Do you actually have the snippet in the theme? The error is saying that  `snippets/testliquid.liquid` does not exist on the current theme you are trying to use.

**Reply by ingalesachin7** [reply]
yes.That is in the theme but it checking in the app extension folder...I want to pass it from theme.. any solution?

**Reply by Kyon147** [reply]
It might be that app blocks are only able to access their own content, so you'd need to put the other file `testliquid` in the same directory, as app blocks are not installed within a theme which is probably why you can't access it.
